tests/unittest.c

```diff
@@ -37,9 +37,17 @@ int dumper(void* user, const char* section, const char* name,
     }
 
 #if INI_HANDLER_LINENO
+# if INI_ALLOW_NO_VALUE
+    printf("... %s%s%s;  line %d\n", name, value ? "=" : "", value ? value : "", lineno);
```

Comment:
Why don't we simplify this by avoiding the `#if INI_ALLOW_NO_VALUE` and just use the line above in all cases? (Value will never be NULL if INI_ALLOW_NO_VALUE is off anyway.) Same below.

Reply:
Changed this one, but GitHub doesn't seem to catch up. Thanks.